session.go

```diff
@@ -604,7 +604,6 @@ func Connect(ctx context.Context, opts ...ConnectOption) (Session, error) {
 		if err != nil {
 			remote := false
 			if resp.Error != "" {
-				err = errors.New(resp.Error)
```

Comment:
This was an oopsie on my part - the internal Session is already setting the `err` for us.